### PR TITLE
P2071R2 Named universal character escapes

### LINKED ISSUE #5576: [2022-07 CWG Motion 11] P2071R2 Named universal character escapes
[P2071R2](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2071r2.html)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes  #5576
Fixes cplusplus/papers#798

### CODE REVIEW

File: source/lex.tex [hunk 1]
```diff
@@ -296,15 +315,135 @@
...
+\unicode{001c}{information separator four} \\
+\unicode{001c}{file separator} \\
+\unicode{001d}{information separator three} \\
+\unicode{001d}{group separator} \\
+\unicode{001e}{information separator two} \\
+\unicode{001e}{record separator} \\
+\unicode{001f}{information separator one} \\
+\unicode{001f}{unit separator} \\
+\columnbreak
+\unicode{0082}{break permitted here} \\
```
Comment (JohelEGP) [inline source/lex.tex:403]: U+007F | DELETE
-- | --


is missing between these two.

**Review by JohelEGP — CHANGES_REQUESTED** [review]

**Comment by Mick235711** [comment]
Don't we already have #5368 for this...?

**Comment by jensmaurer** [comment]
> Don't we already have #5368 for this...?

We do.

File: source/lex.tex (hunk 1 above)
Comment (jensmaurer) [reply source/lex.tex]: Fixed.

**Pull request merged**